### DISCUSSION TITLE
Regarding Raw Input

### DISCUSSION THREAD

**Discussion by StrageFaithApernum2019** [ppy/osu#17872] [Ideas] [answered]
Sorry for the sudden question however if it is ok to ask

Upon reinstalling Osu Lazer I suddenly realized shortly there was no Raw input for the Mouse Settings

I have been using Raw since the Old Original [Soon to be replaced by Osu Lazer] and has been training using the same setting since this very day

I hope Raw Input can be added by then because if not so i will need to relearn the mechanics again without an option of Raw Input

I have recently used a newer Wireless mouse which is adaptable to Raw Input so it would have been easier if I had not used Raw Input as a Setting in the Old Original

Thank You I hope to hear a response soon,
STRage FAITH APERnum

**Comment by bdach** [comment, marked as answer]
raw input is now called "high precision mouse". it even shows up if you search for "raw input" in settings.
